dojo/tools/pwn_sast/parser.py

```diff
@@ -0,0 +1,97 @@
+import json
+import hashlib
+
+from dojo.models import Finding
+
+
+class PWNSASTParser(object):
+    """
+    A class that can be used to parse pwn_sast source code scanning results in JSON format.  See https://github.com/0dayinc/pwn for additional details.
+    """
+
+    def get_scan_types(self):
+        return ["PWN SAST"]
+
+    def get_label_for_scan_types(self, scan_type):
+        return scan_type
+
+    def get_description_for_scan_types(self, scan_type):
+        return "Import pwn_sast Driver findings in JSON format."
+
+    def get_findings(self, filename, test):
+
+        results = json.load(filename)
+        report_name = results["report_name"]
+        data_arr = results["data"]
+
+        findings = {}
+
+        for data_hash in data_arr:
+            timestamp = data_hash["timestamp"]
+            security_requirements = data_hash["security_requirements"]
+            sast_module = security_requirements["sast_module"]
+            section = security_requirements["section"]
+            nist_800_53_uri = security_requirements["nist_800_53_uri"]
+            cwe_id = security_requirements["cwe_id"]
+            cwe_uri = security_requirements["cwe_uri"]
+            filename_hash = data_hash["filename"]
+            git_repo_root_uri = filename_hash["git_repo_root_uri"]
+            offending_file = filename_hash["entry"]
+            line_no_and_contents = data_hash["line_no_and_contents"]
+            test_case_filter = data_hash["test_case_filter"]
```

Comment:
Please use the `.get` notation as it is more forgiving (returns None instead of an exception) when encountering a field that is not in the file